pkg/policyfilter/state.go

```diff
@@ -222,6 +218,25 @@ func (pol *policy) podInfoMatches(pod *podInfo) bool {
 	return pol.podMatches(pod.namespace, pod.labels)
 }
 
+func (pol *policy) containerMatches(container *containerInfo) bool {
+	containerFilterFields := labels.Labels{
+		"name": container.name,
+	}
+	return pol.containerSelector.Match(containerFilterFields)
+}
+
+// get a slice of container cgroupIDs that match the policy
+func (pol *policy) matchingContainersCgroupIDs(containers []containerInfo) []CgroupID {
```

Comment:
Can you add some explanation in the commit message about how the implementation of the filtering?
How about something like:

> The actual filtering happens in the matchingContainersCgroupIDs() function. Whenever a list of cgroup ids is needed (typically to update policyfilter map), we use this function that returns cgroup ids after applying any containerSelector filtering.


Feel free to change above if you feel is wrong, it's just my understanding from reading the code.

Reply:
Thank you for pointing this out! I added explanations for `containerMatches` and `matchingContainersCgroupIDs`.
Hopefully, not it makes more sense:)

Reply:
Also, just wanted to say that I updated docs in the same commit (I updated it) because I noticed a place where I forgot to mention the new filtering mechanism.